Graphics/GraphicsEngineD3D11/src/DeviceContextD3D11Impl.cpp

```diff
@@ -1,2501 +1,2511 @@
-    VERIFY(pInputLayout == m_CommittedD3D11InputLayout, "Inconsistent input layout");
-
-    const Uint32  MaxVBs = D3D11_IA_VERTEX_INPUT_RESOURCE_SLOT_COUNT;
-    ID3D11Buffer* pVBs[MaxVBs];
-    UINT          Strides[MaxVBs];
-    UINT          Offsets[MaxVBs];
-    m_pd3d11DeviceContext->IAGetVertexBuffers(0, MaxVBs, pVBs, Strides, Offsets);
-    auto NumBoundVBs = m_NumCommittedD3D11VBs;
-    for (Uint32 Slot = 0; Slot < MaxVBs; ++Slot)
-    {
-        if (Slot < NumBoundVBs)
-        {
-            const auto& BoundD3D11VB  = m_CommittedD3D11VertexBuffers[Slot];
-            auto        BoundVBStride = m_CommittedD3D11VBStrides[Slot];
-            auto        BoundVBOffset = m_CommittedD3D11VBOffsets[Slot];
-            if (BoundD3D11VB && !pVBs[Slot])
-                VERIFY(pVBs[Slot] == nullptr, "Missing D3D11 buffer detected at slot ", Slot);
-            if (!BoundD3D11VB && pVBs[Slot])
-                VERIFY(pVBs[Slot] == nullptr, "Unexpected D3D11 buffer detected at slot ", Slot);
-            if (BoundD3D11VB && pVBs[Slot])
-            {
-                VERIFY(BoundD3D11VB == pVBs[Slot], "Vertex buffer mismatch detected at slot ", Slot);
-                VERIFY(BoundVBOffset == Offsets[Slot], "Offset mismatch detected at slot ", Slot);
-                VERIFY(BoundVBStride == Strides[Slot], "Stride mismatch detected at slot ", Slot);
-            }
-        }
-        else
-        {
-            VERIFY(pVBs[Slot] == nullptr, "Unexpected D3D11 buffer detected at slot ", Slot);
-        }
-
-        if (pVBs[Slot])
-            pVBs[Slot]->Release();
-    }
-}
-
-template <typename TD3D11ShaderType, typename TGetShaderMethodType>
-void dbgVerifyCommittedShadersHelper(SHADER_TYPE                      ShaderType,
-                                     const CComPtr<ID3D11DeviceChild> BoundD3DShaders[],
-                                     ID3D11DeviceContext*             pCtx,
-                                     TGetShaderMethodType             GetShaderMethod)
-{
-    RefCntAutoPtr<TD3D11ShaderType> pctxShader;
-    (pCtx->*GetShaderMethod)(&pctxShader, nullptr, nullptr);
-    const auto& BoundShader = BoundD3DShaders[GetShaderTypeIndex(ShaderType)];
-    VERIFY(BoundShader == pctxShader, GetShaderTypeLiteralName(ShaderType), " binding mismatch detected");
-}
-void DeviceContextD3D11Impl::dbgVerifyCommittedShaders()
-{
-#    define VERIFY_SHADER(NAME, Name, N) dbgVerifyCommittedShadersHelper<ID3D11##Name##Shader>(SHADER_TYPE_##NAME, m_CommittedD3DShaders, m_pd3d11DeviceContext, &ID3D11DeviceContext::N##SGetShader)
-    // These shaders which are not set will be unbound from the D3D11 device context
-    VERIFY_SHADER(VERTEX, Vertex, V);
-    VERIFY_SHADER(PIXEL, Pixel, P);
-    VERIFY_SHADER(GEOMETRY, Geometry, G);
-    VERIFY_SHADER(DOMAIN, Domain, D);
-    VERIFY_SHADER(HULL, Hull, H);
-    VERIFY_SHADER(COMPUTE, Compute, C);
-}
-
-#endif // VERIFY_CONTEXT_BINDINGS
-} // namespace Diligent
+/*
```

Comment:
Is this line endings?

Reply:
yes

Reply:
it would be better to add .gitattributes with `* text=auto eol=lf`